sunpy/visualization/animator/base.py

```diff
@@ -52,22 +52,32 @@ class BaseFuncAnimator:
     colorbar: `bool`, optional
         Plot a colorbar. Defaults to `False`.
     button_labels: `list`, optional
-        A list of strings to label buttons. Defaults to `None`.
+        A list of strings to label buttons. Defaults to `None`, if not
```

Comment:
```suggestion
        A list of strings to label buttons. Defaults to `None`. If `None`
```
I think this makes more sense